docs/data-sources/alerts.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: "wavefront"
+page_title: "Wavefront: Alerts"
+description: |-
+Get the information about all Wavefront alerts.
+---
+
+# Data Source: wavefront_alerts
+
+Use this data source to get information about all Wavefront alerts.
+
+## Example Usage
+
+```hcl
+# Get the information about all alerts.
+data "wavefront_alerts" "example" {
+}
+```
+
+## Attribute Reference
+
+* `alerts` - List of all alerts in Wavefront. For each alert you will see a list of attributes.
+  * `name` - The name of the alert as it is displayed in Wavefront.
+  * `id` - The ID of the alert in Wavefront.
+  * `alert_type`- The type of alert in Wavefront.
+  * `additional_information` - User-supplied additional explanatory information about this alert.
+  * `target` - A email address or integration endpoint (such as PagerDuty or webhook) to notify when the alert status changes.
```

Comment:
An email address or integration endpoint (such as PagerDuty or webhook) to notify when the alert status changes.